borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+
+```
+navigator.permissions.query({name:'window-placement'})
+  .then((status) => {
+    // Do what you need with the permission state.
+    console.log(status.state)
+  });
+```
+
+#### Disabling Window Management
+
+To disable borderless mode, the user can disable the Window Management
+permission in App Settings.
+
+In case the Window management permission is not provided or no longer in place,
+the app will follow the normal display mode hierarchy based on the
+`display_override` and display values on the `manifest.json`. In case no other
+display mode is provided, it will
+[default to standalone mode](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/web_applications/web_app_proto_utils.cc;drc=4a8573cb240df29b0e4d9820303538fb28e31d84;l=161)
+similarly to WCO and other `display_override` modes.
+
+### 5. Isolated Web App check
+
+Borderless mode is only available for
+[Isolated Web Apps (IWAs)](https://github.com/WICG/isolated-web-apps).
+
+## Related features
+
+- [app-details feature](crbug.com/1225871) (Googlers-only)
+- [go/cros-privacy-indicators-design](go/cros-privacy-indicators-design)
+  (Googlers-only)
+- [additional windowing controls](https://github.com/ivansandrk/additional-windowing-controls/blob/main/awc-explainer.md)
+
+### Displaying app’s origin in the app settings UI
+
+Since without the title bar and the 3-dot menu, the app’s origin won’t be visible
+anymore, it needs to be shown somewhere, in this case in App Settings. This
+work has been completed as part of the [app-details feature](crbug.com/1225871),
+but it is pending its launch.
+
+Displaying the origin is mainly necessary for debugging purposes. Until the app
+details get launched, an alternative way to get the origin of an IWA in
+borderless mode is via the developer console with
+[`document.location.href`](https://developer.mozilla.org/en-US/docs/Web/API/Location/href).
+
+In the future when app details have been launched and origin is available, the
+app settings can be accessed via opening the system’s settings and then looking
+for the right app or by right clicking the app’s icon on the shelf. Or by
+visiting the app's OS-settings in the following type of URL:
+<span style="text-decoration:underline;">chrome://os-settings/app-management/detail?id=&lt;APP_ID></span>
+
+Mock of app settings UI with app’s origin:  
+![App settings UI mock](./images/app-settings-ui.png)
+
+### Privacy indicators
+
+Removing the title bar also removes the area where critical privacy indicators
+would live, e.g. for camera and microphone access. However there is already
+another on-going project to move those privacy indicators to the shelf.
+
+![Example privacy indicator](./images/privacy-indicators-in-shelf.png)
+
+See (Googlers only)
+[go/cros-privacy-indicators-design](go/cros-privacy-indicators-design) for more.
+
+## Other notable details
+
+### “Settings and more” three-dot button
```

Comment:
This isn't necessarily a three-dot button. Officially, the standards do not require any sort of app settings, though many implementations have some kind of app settings menu.

Note that other platforms (like Android) have already solved this by moving some of the settings into the OS notifications tray, so this is not unprecedented. You could cite that precedent here.

Just say "some implementations of PWAs, e.g. Chrome and Edge, show a three-dot menu button (which gives users access to ....)". And then say user agents in borderless mode will want to find some other place to put these, or developers need to understand that they won't have those options.